### ISSUE TITLE
[Bug] Since Basset Update to 1.3.x - Datatable Spinner on every CRUD Page

### ISSUE THREAD

**Issue by sebiecker** [Laravel-Backpack/theme-coreuiv2#37]
# Bug report

### What I did

Update Basset from 1.2.4 to 1.3.x (1.3.2)

### What I expected to happen

Spinner not visible on every CRUD Page

### What happened

Spinner.svg is visible on every CRUD Page
![image](https://github.com/Laravel-Backpack/theme-coreuiv2/assets/8569511/88e2261f-d5d7-47b8-b505-d4917c1c03f0)

### What I've already tried to fix it

Downgrade to Basset 1.2.4 -> no spinner
Switch Theme to Tabler -> no spinner, but i´m not able to switch Themes now.
Switch Theme to CoreUIv4 -> spinner visible

### Context to replicate the bug - Backpack, Laravel, PHP, DB version

When I run ```php artisan backpack:version``` the output is:

### PHP VERSION:
8.3.6

### PHP EXTENSIONS:
Core, date, libxml, openssl, pcre, zlib, filter, hash, json, pcntl, random, readline, Reflection, SPL, session, standard, sockets, bcmath, bz2, calendar, ctype, curl, dom, mbstring, fileinfo, ftp, gettext, iconv, intl, ldap, exif, mysqlnd, PDO, Phar, posix, shmop, SimpleXML, sqlite3, sysvmsg, sysvsem, sysvshm, tokenizer, xml, xmlwriter, xsl, mysqli, pdo_mysql, pdo_sqlite, xmlreader, zip, igbinary, msgpack, redis, xdebug

### LARAVEL VERSION:
11.3.1.0

### BACKPACK PACKAGE VERSIONS:
backpack/basset: 1.3.2
backpack/crud: 6.7.7
backpack/generators: v4.0.5
backpack/logmanager: v5.0.1
backpack/pro: 2.1.12
backpack/theme-coreuiv2: 1.2.3
backpack/theme-tabler: 1.2.8

**Comment by karandatwani92** [comment]
Hey @sebiecker
Sorry for the inconvenience, and thanks for reporting. I'm assigning my colleague @pxpm to fix it.

**[BUG - Details]**
When we update Basset to `1.3.2` Core UI v2 Breaks. Other themes are working fine.

**Comment by pxpm** [comment]
Hey @sebiecker thanks for the heads up and sorry for the bad experience.

This was an unintended side-effect that we didn't spotted in the "default" tabler theme. 

TLDR; we added support in basset to output multiple file formats, including svg. Since we use a little trick here to internalize the datatable spinner, we need to don't output the spinner when we add it to basset. 

I've just released a fix in https://github.com/Laravel-Backpack/CRUD/releases/tag/6.7.8

Don't forget to clear your view caches. And if you are overwriting `datatables_logic.blade.php` in your application, you need to manually apply the fix. 

I will be closing this issue, please re-open if you are still experiencing issues after the update. 

Cheers
